### ISSUE TITLE
Image resizing should be operable with a keyboard

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3977]
Big thanks to Jackie D’Elia (@jdelia) who reported this during WCUS contributor's day.

Any action available in the editor should be operable also using only the keyboard.

When an image is selected, 4 "resize handlers" appear at the image 4 corners:

![screen shot 2017-12-13 at 17 06 50](https://user-images.githubusercontent.com/1682452/33948922-72ab86dc-e028-11e7-82c3-4563f8b77ab2.png)

Clicking and dragging one of these handlers allows to resize the image (the ratio is preserved). Note this doesn't change the actual image size, it just shrinks the image within the browser and sets some CSS.

However, there's no way to resize the image using only the keyboard. At the very least, there should be an alternative way to do it, for example a couple input fields in the sidebar block settings, to allow users to enter the desired width/height. This would also allow all users to "fine-tune" the image size to the exact desired size. i'm not sure what unit measure should be used though (pixels? percentage?)

It would also be nice to add a "Reset" control, to bring back the image to its original size.

**Comment by SuperGeniusZeb** [comment]
There are now input controls for resizing images in the block inspector for the Image block, including some presets and a <kbd><samp>Reset</samp></kbd> button. The only thing that seems to be missing is a way to set just the height or just the width of the image and have the other one automatically change to match. However, that may belong in a separate issue, so should this issue be closed now?

**Comment by afercia** [comment]
Yep I think this issue can be closed. @SuperGeniusZeb would you like to open a new issue for the "preserve ratio" thing? I've also noticed the width/height fields are a bit buggy:
- click on width (initially set to 150), use the up arrow key to increase the value: it starts increasing but not from the current value, it first resets to 0 and then increases
- both fields accept negative values 😬

**Comment by SuperGeniusZeb** [comment]
@afercia I just created issues for all of those problems: #7636, #7637, and #7638.
